src/main/webapp/app/shared/util/utils.tsx

```diff
@@ -116,3 +118,36 @@ export const getEntityTableActionsColumn = (entityType: ENTITY_TYPE) => {
   };
   return actionsColumn;
 };
+
+export function getUserFullName(user: IUser) {
+  let name;
+  if (user.firstName && user.lastName) {
+    name = `${user.firstName} ${user.lastName}`;
+  } else if (user.firstName) {
+    name = user.firstName;
+  } else {
+    name = user.lastName;
+  }
+  return name;
+}
+
+export function getMostRecentComment(comments: Comment[]) {
+  let latestComment = comments[0];
+  for (const comment of comments) {
+    if (parseInt(comment.date, 10) > parseInt(latestComment.date, 10)) {
+      latestComment = comment;
+    }
+  }
+  return latestComment;
+}
```

Comment:
This should be moved to firebase-utils.ts, let's create it if not available. cc @calvinlu3